packages/components/src/Button/Button/Button.module.scss

```diff
@@ -33,47 +33,50 @@
     text-decoration: none;
   }
 
-  &:not(:disabled) {
+  &:not(.disabled, .working) {
     &:active {
       transform: translateY(1px);
     }
+  }
 
-    &:focus {
-      outline: none;
-    }
+  &:focus {
+    outline: none;
+  }
 
-    &:focus-visible {
-      &::after {
-        $focus-ring-offset: calc(
-          (#{$border-focus-ring-border-width} * 2) + 1px
-        );
-
-        content: "";
-        position: absolute;
-        background: transparent;
-        border-color: $color-blue-500;
-        border-radius: $border-focus-ring-border-radius;
-        border-width: $border-focus-ring-border-width;
-        border-style: $border-focus-ring-border-style;
-        inset: calc(-1 * #{$focus-ring-offset});
-      }
+  &:focus-visible {
+    &::after {
+      $focus-ring-offset: calc((#{$border-focus-ring-border-width} * 2) + 1px);
+
+      content: "";
+      position: absolute;
+      background: transparent;
+      border-color: $color-blue-500;
+      border-radius: $border-focus-ring-border-radius;
+      border-width: $border-focus-ring-border-width;
+      border-style: $border-focus-ring-border-style;
+      inset: calc(-1 * #{$focus-ring-offset});
     }
   }
 
   &.reversed {
-    &:not(:disabled) {
-      &:focus-visible {
-        &::after {
-          border-color: $color-blue-300;
-        }
+    &:focus-visible {
+      &::after {
+        border-color: $color-blue-300;
       }
     }
   }
 
-  &:disabled,
+  &.disabled,
   &.working {
-    pointer-events: none;
```

Comment:
We were relying on this as a way of actually stopping the button from triggering its `onClick` when it's in the `working` state, this introduces a bug

Reply:
good context, I've disabled `onClick` and `onPointerDown` on the working button. Kept `onBlur` and `onFocus` and similar if that makes sense